pkg/db/v1beta1/postgres/postgres_test.go

```diff
@@ -18,6 +18,7 @@ package postgres
 
 import (
 	"fmt"
+	"github.com/google/go-cmp/cmp"
```

Comment:
```suggestion
```

Could you move it to the third group? You can verify import orders to perform make lint on your local.

Reply:
Sure, update based on the lint.